test/gui/shared/scripts/pageObjects/SyncConnection.py

```diff
@@ -108,10 +100,12 @@ def unselect_folder_in_selective_sync(folder_name):
                         # example: folder1 (13 B) => folder1
                         item_name = item.text.rsplit(" ", 2)[0]
                         if item_name == folder_name:
+                            # NOTE: checkbox does not have separate object
+                            # click on (11,11) which is a checkbox to unselect the folder
                             squish.mouseClick(
                                 item,
-                                9,
-                                9,
+                                11,
+                                11,
                                 squish.Qt.NoModifier,
                                 squish.Qt.LeftButton,
```

Comment:
@TheOneRing Squish cannot select `checkbox`, `icon` and `text` separately but shows as a single element. Something to fix?

Folder Sync Connection dialog:
![Screenshot from 2024-05-30 14-54-27](https://github.com/owncloud/client/assets/52366632/5a73567c-2f9e-43df-8787-661eb1985e97)